enumerable_methods.rb

```diff
@@ -0,0 +1,65 @@
+module Enumerable
+  def my_each
+    return unless block_given?
+
+    for i in self
+      yield i
+    end
+  end
+
+  def my_each_with_index
+    return unless block_given?
+
+    index = 0
+    for i in self
+      yield i, index
+      index += 1
+    end
+  end
+
```

Comment:
- [ ] enumerables #my_each_with_index returns an enumerator if no block is given
     Failure/Error: expect(array.my_each_with_index).to be_an(Enumerator)